packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -43,12 +39,10 @@ const vercelCreateDomain = async (domain: string) => {
   const data = await response.json();
 
   // Domain is already owned by another team but you can request delegation to access it
-  if (data.error?.code === "forbidden")
-    throw new TRPCError({ code: "CONFLICT", message: "domain_taken_team" });
+  if (data.error?.code === "forbidden") return false;
 
   // Domain is already being used by a different project
-  if (data.error?.code === "domain_taken")
-    throw new TRPCError({ code: "CONFLICT", message: "domain_taken_project" });
+  if (data.error?.code === "domain_taken") return false;
```

Comment:
We don't want to fail creating the org because there was an error with Vercel the same way we want to support other providers which is why a custom DNS missing check was added, in order to let the admins keep track which orgs are effectively well configured.